services/voice_service/README.md

```diff
@@ -0,0 +1,3 @@
+GPU RAM = 1Gb
+cpu time = 0.15 sec 
+gpu time = 0.05 sec 
```

Comment:
is this a relevant info or copied?
Could you please write here a description of th eservice, how it works, what input and output
We now work on readmes, so it would be neccessary anyway

Reply:
Done